package.json

```diff
@@ -9,10 +9,10 @@
   ],
   "scripts": {
     "compile": "hardhat compile",
-    "coverage": "env COVERAGE=true hardhat coverage",
+    "coverage": "cross-env COVERAGE=true hardhat coverage",
     "docs": "npm run prepare-docs && oz-docs",
     "docs:watch": "oz-docs watch contracts docs/templates docs/config.js",
-    "prepare-docs": "scripts/prepare-docs.sh",
+    "prepare-docs": "bash scripts/prepare-docs.sh",
```

Comment:
This script starts with `#!/usr/bin/env bash`. Is it necessary to force bash here? I'm wondering if that may introduce issues on exotic systems ... such as windows machines

Reply:
This entire PR is about making the scripts work on Windows.

Reply:
@Amxx Hello my friend, As @frangio  said, it is necessary to use bash on Windows OS.